rfcs/0004-leader-rotation.md

```diff
@@ -43,7 +43,11 @@ A leader transmits entries during its slot.  After `T` ticks, all the
 validators switch to the next scheduled leader. Validators must ignore entries
 sent outside a leader's assigned slot.
 
-All `T` ticks must be observed from the current leader for that part of PoH to
-be accepted by the cluster. If entries are not observed (leader is down) or
-entries are invalid (leader is buggy or malicious), a validator should fill the
-gap with empty entries and continue with PoH from the next leader.
+All `T` ticks must be observed by the next leader for it to build its own
+entries on. If entries are not observed (leader is down) or entries are invalid
+(leader is buggy or malicious), the next leader must produce empty entries to
```

Comment:
instead of "empty entries", please use "ticks"

Reply:
all slots contain N ticks and any number of non-empty Entries interleaved, only the ticks need to be transmitted to cover the missing/broken leader's slot

I think we should avoid saying "empty entries", as it implies there's something other than `ticks` and `entries` in PoH